js/main.js

```diff
@@ -0,0 +1,8 @@
+// const ele1 = document.querySelector(".span2");
+// const ele2 = document.querySelector("button");
+// const ele3 = document.querySelector("span");
+
+// eslint-disable-next-line no-console
+console.log('first hello world');
+// eslint-disable-next-line no-unused-vars
+const ele3 = document.querySelector('span');
```

Comment:
It seems that you have not implemented any JavaScript functionality for the mobile menu. This is the main requirement of this project.
![image](https://user-images.githubusercontent.com/61361037/210973485-bae14597-723c-4c0e-89d4-02c7e039fb26.png)
Kindly implement the JavaScript functionality.